### PR TITLE
feat(curriculum): adding hotel feedback form workshop

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a9824ac5d9f77ec304969f.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Here's an example of a `textarea` element:
+
+```html
+<textarea id="comments" name="comments" rows="4" cols="50"></textarea>
+```
+
+The `rows` attribute is used to specify the visible height of the `textarea`, and the `cols` attribute is used to specify the visible width of the `textarea`.
+
+Below your `label` element, add a `textarea` element. In the next step you will add the necessary attributes.
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a9824ac5d9f77ec304969f.md:22]:
```suggestion
Below your `label` element, add a `textarea` element. In the next step, you will add the necessary attributes.
```

**Review by Ksound22 — APPROVED** [review]
LGTM 🚀

File: curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a962954f4e0d76223b37ed.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+dashedName: step-23
+---
+
+# --description--
+
+Next, add another `label` element with the text of `Location` and the `for` attribute set to `"location"`. 
+
+For the checkbox `input`, both the `id`, `name` and `value` attributes should be set to `"location"`.
+
+Below that `label` element, add another `label` element with the text of `Reputation` and the for attribute set to `"reputation"`.
```
Comment (moT01) [inline curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a962954f4e0d76223b37ed.md:14]:
```suggestion
Below that `input` element, add another `label` element with the text of `Reputation` and the for attribute set to `"reputation"`. 
```

**Review by moT01 — COMMENTED** [review]
Nicely done @jdwilkin4 🎉 

Found one typo.

When you go to [the registration form](https://www.freecodecamp.dev/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-65), the submit button works (sort of) - it says `Normally this form would be submitted! It works. This will be submitted to: https://register-demo.freecodecamp.org`  Wonder if we should change this so it does something like that. You could change it to something like this: `<form method="post" action="https://hotel-feedback.freecodecamp.org">`

Not necessary, but maybe a nice to have. Add it if you want, otherwise this looks good 👍

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Comment by moT01** [comment]
Hope you don't mind @jdwilkin4, I went ahead and fixed the conflicts here since I had this checked out 👍

**Pull request merged**